cfgov/alerts/post_sqs_messages.py

```diff
@@ -55,26 +55,32 @@
 )
 
 
-def matching_issue(title, issues):
-    return next((issue for issue in issues if issue.title == title), None)
+def post_message(body, title, github_creds={}, mattermost_creds={}):
+    issue = GithubAlert(github_creds).post(
+        title=title,
+        body=body,
+    )
 
+    if mattermost_creds:
+        MattermostAlert(mattermost_creds).post(
+            text='Alert: {}. Github issue at {}'.format(
+                body,
+                issue.html_url,
+            )
+        )
 
-def post_to_chat(endpoint, username, message, issue_url):
-    text = 'Alert: {}. Github issue at {}'.format(
-        message,
-        issue_url,
-    )
-    payload = {
-        'text': text,
-        'username': args.mattermost_username,
-    }
-    requests.post(
-        args.mattermost_webhook_url,
-        data=json.dumps(payload),
+
+def cleanup_jenkins_message(message):
```

Comment:
I still feel like it'd be nice if this happened in the posting-to-sqs instead of the reading-from-sqs, but not necessary to address for the same reasons as before.

In that same line of thought, the `#1234` is going to be a problem regardless of whether the source if Jenkins. Do we want to make that a regex replacement that checks for `r'^#[0-9]+` instead. And we could always expand markdown links. 

Then we could remove `jenkins` from the function name?

Reply:
I'll rename it to `cleanup_message` for now. We can refine it in terms of regexes or move it somewhere else in the next iteration.